migrations/20160901_remove_null_notifications.js

```diff
@@ -0,0 +1,13 @@
+var _ = require('lodash');
+
+db.users.find({}).forEach(function(user){
+  var newNewMessages = {};
+
+  _.each(user.newMessages, function(val, key){
+    if(key != "undefined" && val!=null){
+      newNewMessages[key] = val;
+    }
+  });
+
+  db.users.update({_id: user._id}, {$set: {'newMessages': newNewMessages}});
```

Comment:
@gambtho This command:
`db.users.update({_id: "d904bd62-da08-416b-a816-ba797c9ee265"}, {$unset:{ 'newMessages.4fc50931-0d1c-45bf-b658-077af46c55ad': 1 }},{multi:true});`

will turn this:
````
    "newMessages": {
        "4fc50931-0d1c-45bf-b658-077af46c55ad": null,
        "85dd1be6-336b-4e53-9949-2c09150b4035": {
            "name": "Alys's private guild",
            "value": true
        }
    },
````
into this:
````
    "newMessages": {
        "85dd1be6-336b-4e53-9949-2c09150b4035": {
            "name": "Alys's private guild",
            "value": true
        }
    },
````